packages/react/src/FormErrorList/useAddErrorCountToPageTitle.tsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from 'react'
+
+export const useAddErrorCountToPageTitle = (
+  /** The list of error messages used to calculate the error count. */
+  errors: { id: string; label: string }[],
+  /** The text following the error count. */
+  suffix = {
+    plural: 'invoerfouten',
+    singular: 'invoerfout',
+  },
+) => {
+  const [documentTitle, setDocumentTitle] = useState<string>()
+
+  useEffect(() => {
+    setDocumentTitle(document.title)
+  }, [])
+
+  if (documentTitle) {
```

Comment:
Is it possible in this hook to `return null` if `documentTitle` is nullish to avoid if-statement nesting?

Reply:
Do you mean an early return? Yeah, that is a bit cleaner